apps/web/src/lib/components/SideNavigation.svelte

```diff
@@ -0,0 +1,236 @@
+<script lang="ts">
+	import { AuthService } from '$lib/auth/authService';
+	import { getContext } from '@gitbutler/shared/context';
+	import { env } from '$env/dynamic/public';
+
+	const authService = getContext(AuthService);
+	const token = $derived(authService.token);
+
+	function logout() {
+		authService.clearToken();
+		window.location.href = `${env.PUBLIC_APP_HOST}cloud/logout`;
+	}
+
+	function login() {
+		console.log(env.PUBLIC_APP_HOST);
+		window.location.href = `${env.PUBLIC_APP_HOST}cloud/login`;
```

Comment:
Same here, also left-over `console.log`?

Reply:
That's a very important `console.log` 😂

Reply:
actually, this component is also not used - I did it trying to recreate the sidebar used in the client mockups, but realized that a sidebar on web is maybe not what we really want. anyhow, taking out the log, but also we may want to remove this entire file soon too.